app/models/import_service_config/nemo.rb

```diff
@@ -109,6 +111,9 @@ def populate_study_file(scp_study_id)
         library = load_study&.[]('technique')
       end
       study_file.expression_file_info.library_preparation_protocol = find_library_prep(library)
+      http_url = file_access_info(protocol: :http)&.[]('url')
```

Comment:
In the study created from an HCA h5ad called via API, when I went to the upload wizard, library_preparation_protocol was not populated, do we think we'll have an analogous query in app/models/import_service_config/hca.rb that we'll use to autopopulate `study_file.expression_file_info.library_preparation_protocol` in a later PR? And if NeMO uses a different ontology, some amount of mapping would occur here and in ingest?

Reply:
Yeah it looks like there's a bug with this - I'm looking into it.

Reply:
@jlchang [c227474](https://github.com/broadinstitute/single_cell_portal_core/pull/1955/commits/c2274748e3beb6052d9fd100ecc7d6af708e89b5)